src/components/AnonymousChat/child/ChannelPostNotificationStats.tsx

```diff
@@ -1,12 +1,13 @@
 import * as React from 'react';
 import {Image, StyleSheet, Text, View} from 'react-native';
 
+import IcBlockInactive from '../../../assets/block/Ic_block_inactive';
+import IcComment from '../../../assets/icons/Ic_comment';
+import IcDownvoteOn from '../../../assets/arrow/Ic_downvote_on';
+import IcUpvoteOn from '../../../assets/arrow/Ic_upvote_on';
 import Imageblock from '../../../assets/images/block.png';
-import MemoIc_arrow_down_vote_on from '../../../assets/arrow/Ic_downvote_on';
-import MemoIc_arrow_upvote_on from '../../../assets/arrow/Ic_upvote_on';
-import MemoIc_block_inactive from '../../../assets/block/Ic_block_inactive';
-import MemoIc_comment from '../../../assets/icons/Ic_comment';
 import {BaseChannelItemTypeProps} from '../../../../types/component/AnonymousChat/BaseChannelItem.types';
+import {colors} from '../../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements for the icons have been changed from memoized versions to non-memoized versions. This could potentially impact performance as React will not be able to skip rendering for these components if their props remain the same across renders. Consider reverting back to using the memoized versions of these components.

```diff
- import IcBlockInactive from '../../../assets/block/Ic_block_inactive';
- import IcComment from '../../../assets/icons/Ic_comment';
- import IcDownvoteOn from '../../../assets/arrow/Ic_downvote_on';
- import IcUpvoteOn from '../../../assets/arrow/Ic_upvote_on';
+ import MemoIcBlockInactive from '../../../assets/block/Ic_block_inactive';
+ import MemoIcComment from '../../../assets/icons/Ic_comment';
+ import MemoIcDownvoteOn from '../../../assets/arrow/Ic_downvote_on';
+ import MemoIcUpvoteOn from '../../../assets/arrow/Ic_upvote_on';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->